### PR TITLE
PSMDB-386 Updated previous version variable, added new step to upgrad…

### PR DESCRIPTION
**Pull request by nastena1606**
…e checklists

### CODE REVIEW

**Review by tplavcic — APPROVED** [review]
LGTM

File: source/.res/replace.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@
 .. |mongodb-ce| replace:: MongoDB 4.0 Community Edition
 .. |mongodb| replace:: MongoDB
 .. |percona| replace:: Percona
-.. |prev-version| replace:: 4.0
+.. |prev-version| replace:: 3.6
```
Comment (tplavcic) [inline source/.res/replace.txt:6]: I'm not 100% sure that I know what this does, but are you sure 4.0 should be replaced with 3.6 here - notice you are mergeing to 4.0 branch?

Comment (nastena1606) [reply source/.res/replace.txt:6]: There was a bug with previous version variable (from 4.0 to 4.0). I fixed that.

Comment (tplavcic) [reply source/.res/replace.txt:6]: Cool! Thanks!

**Pull request merged**